Project.toml

```diff
@@ -8,6 +8,7 @@ FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
 GeometryBasics = "5c1252a2-5f33-56bf-86c9-59e7332b4326"
 Joysticks = "82f05805-4863-42dc-b1a7-0852bd62c632"
+KiteModels = "b94af626-7959-4878-9336-2adc27959007"
```

Comment:
KiteViewers should not depend on KiteModels. Instead, it should be a test dependancy.